metaflow/plugins/kfp/tests/setup.cfg

```diff
@@ -0,0 +1,16 @@
+[coverage:run]
+omit = ./*
+
+[coverage:report]
+fail_under = 65
```

Comment:
Could you publish/copy the coverage HTML so we can view what isn't being covered?
```
-- Docs: https://docs.pytest.org/en/stable/warnings.html
----------- coverage: platform linux, python 3.6.8-final-0 -----------
Name                                                                 Stmts   Miss  Cover
----------------------------------------------------------------------------------------
/home/zservice/metaflow/metaflow/plugins/kfp/__init__.py                 0      0   100%
/home/zservice/metaflow/metaflow/plugins/kfp/kfp.py                    178     20    89%
/home/zservice/metaflow/metaflow/plugins/kfp/kfp_cli.py                 70      7    90%
/home/zservice/metaflow/metaflow/plugins/kfp/kfp_constants.py            8      0   100%
/home/zservice/metaflow/metaflow/plugins/kfp/kfp_decorator.py           43     24    44%
/home/zservice/metaflow/metaflow/plugins/kfp/kfp_foreach_splits.py      82     44    46%
/home/zservice/metaflow/metaflow/plugins/kfp/kfp_step_function.py       18     17     6%
/home/zservice/metaflow/metaflow/plugins/kfp/kfp_step_init.py           37     30    19%
----------------------------------------------------------------------------------------
TOTAL                                                                  436    142    67%
Required test coverage of 65.0% reached. Total coverage: 67.43%
```

Reply:
@talebzeghmi where do you want to coverage HTML published?

Reply:
Checkout gitlab publish code coverage. You can find examples of how our cicd does it. It's a command to copy a directory